packages/malloy-db-snowflake/src/snowflake_executor.ts

```diff
@@ -170,10 +170,12 @@ export class SnowflakeExecutor {
 
   private async _setSessionParams(conn: Connection) {
     // set some default session parameters
+    // ensure we do not ignore case for quoted identifiers
```

Comment:
Can you explain why you added this back?  Have you experimented with this?

Reply:
hey @lloydtabb yes I changed this to essentially get tests to pass, commented underneatb